modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -417,8 +417,7 @@ public void setUp() throws Exception {
 						).name(
 							ObjectFieldSettingConstants.NAME_TIME_STORAGE
 						).value(
-							ObjectFieldSettingConstants.
-								VALUE_USE_INPUT_AS_ENTERED
+							ObjectFieldSettingConstants.VALUE_CONVERT_TO_UTC
```

Comment:
Maybe we need to duplicate the date time object fields for both configurations to double check the other behaviour